app/lib/hooks/useVideoConf/StartACallActionSheet.tsx

```diff
@@ -0,0 +1,91 @@
+import { Camera, CameraType } from 'expo-camera';
+import React, { useState } from 'react';
+import { StyleSheet, View } from 'react-native';
+import { useDispatch } from 'react-redux';
+
+import { ESounds, useVideoConfRinger } from '.';
+import { useAppSelector } from '..';
+import { cancelCall, initVideoCall } from '../../../actions/videoConf';
+import AvatarContainer from '../../../containers/Avatar';
+import Button from '../../../containers/Button';
+import { CallHeader } from '../../../containers/CallHeader';
+import i18n from '../../../i18n';
+import { getUserSelector } from '../../../selectors/login';
+import { useTheme } from '../../../theme';
+import useUserData from '../useUserData';
+
+export default function StartACallActionSheet({ rid }: { rid: string }): React.ReactElement {
```

Comment:
- Go to a group
- Start a call
- It's going to "fake" ring for half a second

I'm ok with the "animate to give the impression of creating a call", but it shouldn't ring, nor say "calling"